sphinx/themes/basic/static/doctools.js

```diff
@@ -153,10 +153,7 @@ var Documentation = {
   init : function() {
     this.fixFirefoxAnchorBug();
     this.highlightSearchWords();
-    this.initIndexTable();
```

Comment:
I'm not sure what this does, but why has it been removed?

Reply:
Good points, fixed!